nautilus_core/model/src/data/bar.rs

```diff
@@ -146,13 +149,30 @@ impl Display for BarSpecification {
     feature = "python",
     pyo3::pyclass(module = "nautilus_trader.core.nautilus_pyo3.model")
 )]
-pub struct BarType {
-    /// The bar types instrument ID.
-    pub instrument_id: InstrumentId,
-    /// The bar types specification.
-    pub spec: BarSpecification,
-    /// The bar types aggregation source.
-    pub aggregation_source: AggregationSource,
+pub enum BarType {
+    StandardBar {
```

Comment:
I'd suggest we go for `Standard` and `Composite` as the variant names. Seems cleaner as we know its a type of bar already?